drv/ignition-server/src/main.rs

```diff
@@ -112,10 +190,7 @@ impl idl::InOrderIgnitionImpl for ServerImpl {
         &mut self,
         _: &userlib::RecvMessage,
     ) -> Result<u64, RequestError> {
-        self.controller
```

Comment:
Since this can never fail, you can use `err: ServerDeath` (instead of `err: CLike("IgnitionError")`) in the Idol API, and `Result<u64, RequestError<core::convert::Infallible>>` as the return type.